Utils/Dataflow/009_oracleConnector/Oracle2JSON.py

```diff
@@ -0,0 +1,307 @@
+#!/bin/env python
+import os
+import json
+import argparse
+import ConfigParser
+import re
+import DButils
+import sys
+from datetime import datetime, timedelta
+from collections import defaultdict
+
+try:
+    import cx_Oracle
+except:
+    print "****ERROR : DButils. Cannot import cx_Oracle"
+    pass
+
+# Policies
+PLAIN_POLICY = 'PLAIN'
+SQUASH_POLICY = 'SQUASH'
+
+def connectDEFT_DSN(dsn):
+    connect = cx_Oracle.connect(dsn)
```

Comment:
We should catch exceptions for failed connections.

Reply:
Addressed in PR #80